x/sync/sync_test.go

```diff
@@ -61,11 +61,7 @@ func Test_Creation(t *testing.T) {
 	db, err := merkledb.New(
 		context.Background(),
 		memdb.New(),
-		merkledb.Config{
-			Tracer:        newNoopTracer(),
-			HistoryLength: 0,
```

Comment:
doesn't this change the test now though? IE HistoryLength: 0 vs HistoryLength: 2048 or is that not relevant for the test case?

Reply:
well it passes :)